count_server/web/views/session_view.ex

```diff
@@ -0,0 +1,15 @@
+defmodule CountServer.SessionView do
+  use CountServer.Web, :view
+
+  def render("new.json", %{reply: reply}) do
+    reply
+  end
+
```

Comment:
I think you can write this in one line:
def render("new.json", %{reply: reply}), do: reply